scss/_badges.scss

```
@@ -26,4 +26,10 @@
   &-ui {
     background-color: $ui;
   }
+  &-ui-light {
+    background-color: $ui-light;
+  }
```

This should be the same as the light contrast colors and also have the text be dark.
```suggestion
  &-ui-light {
    background-color: $ui-light;
    color: $black;
  }
```